Source/NSURLSession.m

```diff
@@ -921,6 +921,8 @@ - (void)dealloc
 
 #if	defined(HAVE_DISPATCH_CANCEL)
   dispatch_cancel(_timer);
+#else
```

Comment:
We can just use `dispatch_source_cancel` without the ifdef. 

The timer is under the `dispatch_source` prefix: https://developer.apple.com/documentation/dispatch/1385606-dispatch_source_set_timer